docs/reference/commandline/info.md

```diff
@@ -21,6 +21,7 @@ For example:
     $ docker -D info
     Containers: 14
     Images: 52
+    Engine Version: 1.8.0
```

Comment:
Just a "nit": could you use `1.9.0` in the examples (given that this feature won't be available until docker 1.9.0)